Source/JavaScriptCore/assembler/LinkBuffer.h

```diff
@@ -111,17 +111,21 @@ class LinkBuffer {
 #undef COUNT_LINKBUFFER_PROFILE
     static constexpr unsigned numberOfProfilesExcludingTotal = numberOfProfiles - 1;
 
-    LinkBuffer(MacroAssembler& macroAssembler, void* ownerUID, Profile profile = Profile::Uncategorized, JITCompilationEffort effort = JITCompilationMustSucceed)
-        : m_ownerUID(ownerUID)
-        , m_profile(profile)
+    LinkBuffer(MacroAssembler& macroAssembler, [[maybe_unused]] void* ownerUID, Profile profile = Profile::Uncategorized, JITCompilationEffort effort = JITCompilationMustSucceed)
+        : m_profile(profile)
     {
+#if OS(LINUX) || OS(DARWIN)
```

Comment:
I'm curious, why isn't this used on your platform?

Reply:
I'm going to split this part into another PR.